### DISCUSSION TITLE
Desperate offering

### DISCUSSION THREAD

**Discussion by LouderthanThunder** [Lambosaurus/hives-and-torches#29] [Feats]
You may deal a single wound to yourself as your reaction during your turn. The dm allocates a flesh wound to your character and you suffer all side effects. You may regain a single spell slot of your choice 

Only to be taken by divine casters who worship an approving deity

**Comment by Lambosaurus** [comment]
Heheheh. I love it.

Maybe rename it "Blood offering"?

Again, I worry that its such a situational action its not very competitive with other feats. Could we add another perk?

**Reply by LouderthanThunder** [reply]
Depends what level spell it gives you. Desperate is more open to player rp

Could make it an either/or

**Reply by Lambosaurus** [reply]
Fair about the naming.
